opencolorio_config_aces/config/reference/generate/config.py

```diff
@@ -547,98 +696,172 @@ def generate_config_aces(config_name=None,
-                        ACES_CONFIG_OUTPUT_ENCODING_COLORSPACE):
-                continue
-
-            colorspace = node_to_colorspace(graph, node, complete_description)
-
-            family_colourspaces.append(colorspace)
-
-            if family == 'output_transform':
-                display = beautify_display_name(
-                    node_to_ctl_transform(graph, node).genus)
-                displays.add(display)
-                view = beautify_view_name(colorspace.getName())
-                views.append({
-                    'display': display,
-                    'view': view,
-                    'colorspace': colorspace.getName()
+    for style, transforms_data in config_mapping.items():
+        if transforms_data[0]['interface'] == 'ViewTransform':
+            view_transform = style_to_view_transform(style, [
+                transform_data['ctl_transform']
+                for transform_data in transforms_data
+            ], describe)
+            view_transforms.append(view_transform)
+            view_transform_name = view_transform.getName()
+            view_transform_names.append(view_transform_name)
+
+            for transform_data in transforms_data:
+                display_style = transform_data[
+                    'linked_display_colorspace_style']
+
+                display = style_to_display_colorspace(display_style)
+                display_name = display.getName()
+
+                if display_name not in display_names:
+                    displays.append(display)
+                    display_names.append(display_name)
+
+                shared_views.append({
+                    'display': display_name,
+                    'view': view_transform_name,
+                    'view_transform': view_transform_name,
                 })
-
-            if additional_data:
-                colorspaces_to_ctl_transforms[colorspace] = (
-                    node_to_ctl_transform(graph, node))
-
-        colorspaces += family_colourspaces
-
-    displays = sorted(list(displays))
-    displays.insert(0, displays.pop(displays.index('sRGB')))
-    views = sorted(views, key=lambda x: (x['display'], x['view']))
-
-    # Utility Raw
-    colorspaces.append(
-        colorspace_factory(
-            'Utility - Raw',
-            'Utility',
-            description='The utility "Raw" colorspace.',
-            is_data=True))
-    for display in displays:
-        view = beautify_view_name(colorspaces[-1].getName())
-        views.append({
+        else:
+            for transform_data in transforms_data:
+                ctl_transform = transform_data['ctl_transform']
+                colorspace = ctl_transform_to_colorspace(
+                    ctl_transform,
+                    describe,
+                    to_reference=create_builtin_transform(style))
+
+                colorspaces.append(colorspace)
+
+    no_tonescale_view_transform = view_transform_factory(
+        'Output - No Tonescale',
```

Comment:
Yes!

Reply:
Proposed description string: "This 'identity view' is useful for presenting the source colorimetry directly on the display without any tone-mapping. For example, it may be useful for viewing textures as they would appear in an image editing program. It is not an official ACES transform."

And as Michael mentioned in the meeting, please call setDefaultViewTransformName on the config object to set this as the default view transform.

Reply:
Thanks for changing the name of that view.  Any disagreement on my proposed Description or setting it as the default view transform?